Licensing_Program/userfiles_handling.py

```diff
@@ -118,6 +118,13 @@ def write_header(header_text, user_filepath, insert_linenum=1, cut_lines=0):
                 delete=False) as outfile:
                 for i, line in enumerate(user_file, 1):
                     if i == insert_linenum:
+                        if i == 1:
+                            if line.startswith('#!'):
```

Comment:
I think the if statements on line 121 and 122 can be combined so that there isn't an additional level of indentation.